README.md

```diff
@@ -244,7 +244,7 @@ Tauri is a framework for building desktop mobile apps where your frontend is wri
 
 Leptos is a library for building fullstack web-apps, similar to SolidJS and SolidStart. The two libraries share similar goals on the web, but have several key differences:
 
-- **Reactivity model**: Leptos uses signals to drive both reactivity and rendering, while Dioxus uses signals just for reactivity. For managing re-renders, Dioxus uses a highly optimized VirtualDOM to support desktop and mobile architectures. Both Dioxus and Leptos are extremely fast.
+- **Reactivity model**: Leptos uses signals to drive both reactivity and rendering, while Dioxus uses signals just for reactivity. For managing re-renders, Dioxus uses a highly optimized VirtualDOM to support desktop and mobile architectures. Both Dioxus and Leptos are extremely fast. Under the hood, Dioxus leverages [sledgehammer](https://docs.rs/sledgehammer), a low-level runtime that brings performance very close to Vanilla JS.
```

Comment:
But Dioxus doesn't use sledgehammer anymore. It moved to https://github.com/ealmloff/sledgehammer_bindgen

Reply:
> But Dioxus doesn't use sledgehammer anymore. It moved to https://github.com/ealmloff/sledgehammer_bindgen

Thanks for the review! I know, but I thought it would be better to mention the original project before moving to `sledgehammer_bindgen`.  Lemme know if you'd like me to update it to `sledgehammer_bindgen` or keep it as is.

Reply:
no reason to provide outdated info

Reply:
I just wanted to mention Sledgehammer, as it hasn't been referenced anywhere.

> no reason to provide outdated info

This comment is clearly steering the conversation in a counterproductive direction. Feel free to revise it as you see best fit. I have more important things to focus on 👉👈 than continuing this discussion in this direction.

Reply:
I don't think this should be told to you more than once:

> But Dioxus doesn't use sledgehammer anymore. It moved to https://github.com/ealmloff/sledgehammer_bindgen

Reply:
> I don't think this should be told to you more than once:
> 
> > But Dioxus doesn't use sledgehammer anymore. It moved to https://github.com/ealmloff/sledgehammer_bindgen

Yes, I can read English, I just wanted to give the user a chance to explore the history behind `sledgehammer_bindgen` 👉👈.

Reply:
create a HISTORY.md then, if you want to